client/packages/programs/src/JsonForms/common/JsonForm.tsx

```diff
@@ -152,6 +155,7 @@ const FormComponent = ({
           setError?.(false);
         }
       }}
+      ajv={handleDefaultsAjv}
```

Comment:
@clemens-msupply -- didn't end up using this as we discussed, but do you think it's okay to leave the functionality in place so "default" values can be used when we want them?

Reply:
Should we remove till we need it? might be confusing otherwise

Reply:
I would expect default values to be fairly standard functionality, even if we're not currently utilising it. Have added an explanatory comment: 72247bef8